src/app/widgets/request-wrapper/request-wrapper.component.html

```diff
@@ -0,0 +1,23 @@
+<ng-container *ngIf="request$ | async"></ng-container>
+
+<nz-spin
+  [nzTip]="label"
+  [nzDelay]="delay"
+  [nzIndicator]="indicator"
+  [nzSize]="size"
+  [nzSpinning]="isLoading"
+  [nzSimple]="isSimple"
+>
+  <div *ngIf="error" class="container d-flex flex-column align-items-center content-center gap-8 p-10">
+    <div nz-icon nzType="frown" nzTheme="twotone" nzTwotoneColor="#ff4d4f" [style.font-size.px]="64"></div>
```

Comment:
Major: I think it's better to add style to css since there is no dynamics here

Reply:
Fixed